app/commands.py

```diff
@@ -465,20 +435,19 @@ def migrate_data_to_ft_billing(start_date, end_date):
                         coalesce(n.rate_multiplier,1) as rate_multiplier,
                         s.crown,
                         coalesce((select rates.rate from rates
-                        where n.notification_type = rates.notification_type and n.sent_at > rates.valid_from
+                        where n.notification_type = rates.notification_type and n.created_at > rates.valid_from
                         order by rates.valid_from desc limit 1), 0) as sms_rate,
-                        coalesce((select l.rate from letter_rates l where n.rate_multiplier = l.sheet_count
+                        coalesce((select l.rate from letter_rates l where n.billable_units = l.sheet_count
                         and s.crown = l.crown and n.notification_type='letter'), 0) as letter_rate,
                         coalesce(n.international, false) as international,
                         n.billable_units,
                         1 as notifications_sent
                     from public.notification_history n
-                    left join templates t on t.id = n.template_id
-                    left join dm_datetime da on n.created_at>= da.utc_daytime_start
-                        and n.created_at < da.utc_daytime_end
                     left join services s on s.id = n.service_id
                     where n.notification_status!='technical-failure'
```

Comment:
can remove this tech failure check now that we're matching exact statuses below

Reply:
of course - doh